3_visualize/src/plot_data.R

```diff
@@ -0,0 +1,56 @@
+#This function was written to visualize the trial dataset for learning Targets
+
+#Input: Processed data from second step in targets framework
+#Output: a plot of these data
+
+
+plot_results <- function(eval_data){
+  
+  
+# Create a plot
+  plot_path <- here::here("3_visualize/out/","figure_1.png")
```

Comment:
For all of your functions that write out data, I would recommend you make the filepath an _input_ to the function. That way, you can clearly see from the code used to run the functions what the files are called and where they will live from the top-level without digging into the function code itself. I also don't think `here::here()` is necessary in these instances since all filepaths will always be relative to the repo/project.